app.R

```diff
@@ -39,14 +40,14 @@ if (is.null(data)) {
   ui <- fluidPage(
 
     # CSS
-    includeCSS("sszTheme.css"),
+    includeCSS("sszThemeShiny.css"),
 
     # App Selection
     tags$div(
-      class = "radioDiv",
+      class = "queryDiv",
```

Comment:
compare with zuericssstyle: the function for the radio buttons already wraps the radio buttons in a [radioDiv](https://github.com/StatistikStadtZuerich/zuericssstyle/blob/main/R/ssz_ui_utils.R#L263), so no need to have this here. also I do not see a `queryDiv` anyway

Reply:
We need `queryDiv` only LIMA since the first two Radio Buttons for the initial query should look different than the rest.